test/parallel/test-runner-exit-code.js

```diff
@@ -16,11 +16,11 @@ async function runAndKill(file) {
   });
   const [code, signal] = await once(child, 'exit');
   await finished(child.stdout);
-  assert.strictEqual(code, 1);
-  assert.strictEqual(signal, null);
   if (common.isWindows) {
     common.printSkipMessage('signals are not supported in windows');
```

Comment:
Should we skip the test altogether on Windows? It's pointless to make the test run without testing anything with it.